sources/logdog/Cargo.toml

```diff
@@ -8,10 +8,15 @@ publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
```

Comment:
nit: many of these commit messages say `rustl's` instead of `rustls`, and `FIPs` instead of `FIPS`.

The sentences should all end with a `.` also.